Pod/Classes/VOKCoreDataManager.m

```diff
@@ -93,7 +95,7 @@ - (NSManagedObjectContext *)tempManagedObjectContext
         return nil;
     }
 
-    NSManagedObjectContext *tempManagedObjectContext = [[NSManagedObjectContext alloc] initWithConcurrencyType:NSConfinementConcurrencyType];
+    NSManagedObjectContext *tempManagedObjectContext = [[NSManagedObjectContext alloc] initWithConcurrencyType:NSPrivateQueueConcurrencyType];
```

Comment:
`NSConfinementConcurrencyType` is deprecated as of iOS 9.  We'll need to change to something else.  Maybe we should allow passing in the concurrency type and default to `NSMainQueueConcurrencyType`?